django/db/migrations/state.py

```diff
@@ -588,7 +588,10 @@ def get_index_by_name(self, name):
         raise ValueError("No index named %s on model %s" % (name, self.name))
 
     def __repr__(self):
-        return "<ModelState: '%s.%s'>" % (self.app_label, self.name)
+        return "<%s: '%s.%s'>" % (
```

Comment:
Not for a single line, the idea is to add one for things like multi-line dicts/tuples so that if more lines are added later, we don't have to modify the last line and add a comma.